1.1/jessie/build-msbuild/Dockerfile

```diff
@@ -48,6 +49,10 @@ RUN curl -o /tmp/packagescache.tar.gz \
     && mkdir /packagescache && cd /packagescache \
     && tar xvf /tmp/packagescache.tar.gz \
     && rm /tmp/packagescache.tar.gz \
+    && curl -o /tmp/packagescache.tar.gz \
+    https://dist.asp.net/packagecache/${ASPNETCORE_PATCH_CACHE_VERSION}/aspnetcore.packagecache-${ASPNETCORE_PATCH_CACHE_VERSION}-debian.8-x64.tar.gz \
```

Comment:
Why don't we just regenerate the entire package cache for 1.1.1? E.g. for the 1.0.3 cache, we have the full graph of crossgen-ed dependencies, even if they are 1.0.0 or 1.0.1.